packages/twenty-front/src/modules/favorites/components/FavoriteFoldersMultiSelect.tsx

```diff
@@ -0,0 +1,211 @@
+import { FAVORITE_FOLDERS_DROPDOWN_ID } from '@/favorites/constants/FavoriteFoldersDropdownId';
+import { useFavoriteFoldersScopedStates } from '@/favorites/hooks/useFavoriteFoldersScopedStates';
+import { useMultiFavoriteFolder } from '@/favorites/hooks/useMultiFavoriteFolder';
+import { FavoriteFoldersScopeInternalContext } from '@/favorites/scopes/scope-internal-context/favoritesScopeInternalContext';
+import { isFavoriteFolderCreatingState } from '@/favorites/states/isFavoriteFolderCreatingState';
+import { ObjectRecord } from '@/object-record/types/ObjectRecord';
+import { DropdownMenu } from '@/ui/layout/dropdown/components/DropdownMenu';
+import { DropdownMenuItemsContainer } from '@/ui/layout/dropdown/components/DropdownMenuItemsContainer';
+import { DropdownMenuSearchInput } from '@/ui/layout/dropdown/components/DropdownMenuSearchInput';
+import { DropdownMenuSeparator } from '@/ui/layout/dropdown/components/DropdownMenuSeparator';
+import { useDropdown } from '@/ui/layout/dropdown/hooks/useDropdown';
+import { useScopedHotkeys } from '@/ui/utilities/hotkey/hooks/useScopedHotkeys';
+import { useAvailableScopeIdOrThrow } from '@/ui/utilities/recoil-scope/scopes-internal/hooks/useAvailableScopeId';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { useCallback } from 'react';
+import { useRecoilState, useRecoilValue } from 'recoil';
+import { Key } from 'ts-key-enum';
+import { IconPlus, MenuItem, MenuItemMultiSelect } from 'twenty-ui';
+import { useDebouncedCallback } from 'use-debounce';
+
+const StyledDropdownContainer = styled.div`
+  position: relative;
+  width: 100%;
+`;
+
+const StyledItemsContainer = styled.div`
+  width: 100%;
+  max-height: 160px;
+  overflow-y: auto;
+
+  &::-webkit-scrollbar {
+    display: none;
+  }
+
+  -ms-overflow-style: none;
+  scrollbar-width: none;
+`;
+
+const StyledFooter = styled.div`
+  background: ${({ theme }) => theme.background.primary};
+  border-bottom-left-radius: ${({ theme }) => theme.border.radius.md};
+  border-bottom-right-radius: ${({ theme }) => theme.border.radius.md};
+  border-top: 1px solid ${({ theme }) => theme.border.color.light};
+  bottom: 0;
+  position: sticky;
```

Comment:
this should not be sticky.
We can address this with different containers having max height. No need for stickyness here